ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -216,6 +228,7 @@ export default class DepositAll extends IronfishCommand {
 
   private async verifyCanSend(
```

Comment:
can we take this opportunity to move this entire function into `utils/currency.ts` and only have it one time for both`deposit` and `depositAll`? As far as I can tell, the code is identical.

Reply:
will do